src/app/zap-templates/zcl/data-model/chip/zone-management-cluster.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/ZoneManagement.adoc
+Parameters: in-progress 
+Git: 0.9-fall2024-303-g216988a1d
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="Measurement &amp; Sensing"/>
+  <enum name="StatusCodeEnum" type="enum8">
+    <cluster code="0x0550"/>
+    <item name="ZONE_NOT_FOUND" value="0x02"/>
+    <item name="ZONE_IN_USE" value="0x03"/>
```

Comment:
I had asked in spec @ https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10004/files/54b1ae03586bfccb09710e57062d5a7d05ea4983#r1784922173 but wasn't sure if status codes are exempted from the formatting? (I've noticed every convention seems to have a list of exceptions 😅)

Reply:
> I've noticed every convention seems to have a list of exceptions

Yeah, hence "most other parts".  There are exceptions because sometimes people just made stuff up and no one noticed, and sometimes the issues about parts of the spec not matching other parts got ignored and then we shipped inconsistent things.

Reply:
Regenerated using updated spec with CamelCase